django_cloud_deploy/workflow/__init__.py

```diff
@@ -299,7 +299,8 @@ def create_and_deploy_new_project(
                     300, '[9/{}]: Deployment'.format(self._TOTAL_NEW_STEPS)):
                 app_url = self.deploy_workflow.deploy_gae_app(
                     project_id, django_directory_path, is_new=is_new)
-
+        self._static_content_workflow.set_cors_policy(cloud_storage_bucket_name,
```

Comment:
Maybe a dumb question, but cors is smart enough for all of the different links from the website to work. example. www.website.com is the app_url, but www.website.com/other_page will also work?

Reply:
It should work. I only added https://django-527123.appspot.com in the policy, and when I visit https://django-527123.appspot.com/login/?next=/ the website shows the font correctly.

Reply:
cool beans, just wanted to confirm.